cmd/protoc-gen-crd/main.go

```diff
@@ -72,26 +83,54 @@ func generate(request *plugin.CodeGeneratorRequest) (*plugin.CodeGeneratorRespon
 		}
 	}
 
-	m := protomodel.NewModel(request, true)
+	m := protomodel.NewModel(request, false)
+	channelOutput := map[string]*genMetadata{
+		standardChannelFileName: {
+			shouldGen:           true,
+			includeExperimental: true,
+			fds:                 make([]*protomodel.FileDescriptor, 0),
+		},
+		stableChannelFileName: {
+			shouldGen:           true,
+			includeExperimental: false,
+			fds:                 make([]*protomodel.FileDescriptor, 0),
+		},
+	}
 
-	filesToGen := make(map[*protomodel.FileDescriptor]bool)
 	for _, fileName := range request.FileToGenerate {
 		fd := m.AllFilesByName[fileName]
 		if fd == nil {
 			return nil, fmt.Errorf("unable to find %s", request.FileToGenerate)
 		}
-		filesToGen[fd] = true
+
+		channelOutput[standardChannelFileName].fds = append(channelOutput[standardChannelFileName].fds, fd)
+		// We'll later remove the files from the stable channel that are experimental
+		channelOutput[stableChannelFileName].fds = append(channelOutput[stableChannelFileName].fds, fd)
 	}
 
 	descriptionConfiguration := &DescriptionConfiguration{
 		IncludeDescriptionInSchema: includeDescription,
 	}
 
-	g := newOpenAPIGenerator(
-		m,
-		descriptionConfiguration,
-		enumAsIntOrString)
-	return g.generateOutput(filesToGen)
+	response := plugin.CodeGeneratorResponse{}
+	for outputFileName, meta := range channelOutput {
+		meta := meta
+		g := newOpenAPIGenerator(
+			m,
+			descriptionConfiguration,
+			enumAsIntOrString,
+			meta.includeExperimental,
+		)
+		filesToGen := map[*protomodel.FileDescriptor]bool{}
+		for _, fd := range meta.fds {
+			// All files will be generated in each channel, as shouldGen is true for all
```

Comment:
Are some resource files skipped entirely for the stable channel?

Reply:
Only those resources WasmPlugin and EnvoyFilter that is denoted with `releaseChanel:standard` tag